src/components/cc-toaster/cc-toaster.js

```diff
@@ -6,9 +6,7 @@ import '../cc-toast/cc-toast.js';
 /**
  * @typedef {import('./cc-toaster.types.js').ToastPosition} ToastPosition
  * @typedef {import('./cc-toaster.types.js').ToastAnimation} ToastAnimation
- * @typedef {import('./cc-toaster.types.js').Notification} Notification
  * @typedef {import('./cc-toaster.types.js').ToastOptions} ToastOptions
- * @typedef {import('./cc-toaster.types.js').Toast} Toast
```

Comment:
this type is used in the JS Doc for the `createToast` / `renderToast` methods + for an internal property. Should probably keep the import to let the IDE do its magic.